src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => resolve());
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
```

Comment:
The rejection message 'Promise was rejected!' is hardcoded. Ensure that this message is appropriate for your application's context, or consider making it more descriptive.